kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/SimpleByteBuf.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.connect.s3.storage;
+
+import java.nio.ByteBuffer;
+
+/**
+ * A simple byte buf
+ */
+public class SimpleByteBuf implements ByteBuf {
+
+  private ByteBuffer buffer;
+
+  public SimpleByteBuf(int capacity) {
+    this.buffer = ByteBuffer.allocate(capacity);
+  }
+
+  @Override
```

Comment:
nit - @Override isn't required, we are not extending, but implementing an interface.

Reply:
Done